README.md

```diff
@@ -2,6 +2,9 @@
 
 > Interact with Bitcoin SV ARC exposing the interface [interface.go](https://github.com/bitcoin-sv/go-broadcast-client/blob/main/broadcast/interface.go)
 
+## Documentation
+
+For in-depth information and guidance, please refer to the [SPV Wallet Documentation](https://bsvblockchain.gitbook.io/docs).
```

Comment:
I'm not sure if we should add this section in this repository, as actually in the git book there is not much about go-broadcast-client

Reply:
Should I delete it ?